### DISCUSSION TITLE
Property based

### DISCUSSION THREAD

**Discussion by lsap** [Bogdan17-bit/Trading-Platform#7] [Ideas]
Could you shed a light whether it's feasible to implement or use property based testing tool?

**Comment by Bohdan17-bit** [comment]
Sorry, but I don't know much about testing, so I can't tell you that.
So far, the project has only basic checks for exceptional situations.

**Comment by Bohdan17-bit** [comment]
Isap, what can you recommend in the "edited readme file" topic?
